product/models.py

```diff
@@ -0,0 +1,143 @@
+    is_default     = models.BooleanField(default=False)
+    gender_code    = models.OneToOneField('GenderCode', on_delete=models.CASCADE)
+    age_level      = models.OneToOneField('AgeLevel', on_delete=models.CASCADE)
+    activity_level = models.OneToOneField('ActivityLevel', on_delete=models.CASCADE)
+    care_smoker    = models.BooleanField(default=False)
+    care_drinker   = models.BooleanField(default=False)
+    care_obesity   = models.BooleanField(default=False)
+    disease        = models.ManyToManyField('Disease', through='ProductDisease')
+    allergy        = models.ManyToManyField('Allergy', through='ProductAllergy')
+    dietary_habit  = models.ManyToManyField('DietaryHabit', through='ProductDietaryHabit')
+    goal           = models.ManyToManyField('Goal', through='ProductGoal')
+
+    class Meta:
+        db_table = 'products'
+
+class Image(models.Model):
+    product   = models.ForeignKey('Product', on_delete=models.CASCADE)
+    image_url = models.CharField(max_length=2000)
+
+    class Meta:
+        db_table='images'
+    
+class ProductStock(models.Model):
+    product = models.ForeignKey('Product', on_delete=models.CASCADE)
+    size    = models.CharField(max_length=20, null=True)
+    price   = models.DecimalField(max_digits=10, decimal_places=2)
+    stock   = models.IntegerField()
+
+    class Meta:
+        db_table = 'product_stocks'
+
+class RelatedProduct(models.Model):
+    standard_product = models.ForeignKey('Product', on_delete=models.CASCADE, related_name='related_product')
+    related_product  = models.ForeignKey('Product', on_delete=models.CASCADE, related_name='standard_product')
+
+    class Meta:
+        db_table = 'related_products'
+
+class Disease(models.Model):
+    name = models.CharField(max_length = 30)
+
+    class Meta:
+        db_table = 'diseases'
+
+class ProductDisease(models.Model):
+    product = models.ForeignKey('Product', on_delete=models.CASCADE)
+    disease = models.ForeignKey('Disease', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'product_diseases'
+
+class Allergy(models.Model):
+    name = models.CharField(max_length=30)
+
+    class Meta:
+        db_table='allergies'
+
+class ProductAllergy(models.Model):
+    product = models.ForeignKey('Product', on_delete=models.CASCADE)
+    allergy = models.ForeignKey('Allergy', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table='product_allergy'
+
+class Goal(models.Model):
+    name = models.CharField(max_length = 20)
+    icon = models.CharField(max_length = 20, null = True)
+
+    class Meta:
+        db_table = 'goals'
+
+class ProductGoal(models.Model):
+    product = models.ForeignKey('Product', on_delete=models.CASCADE)
+    goal    = models.ForeignKey('Goal', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'product_goals'
+
+class DietaryHabit(models.Model):
+    name = models.CharField(max_length = 30)
+
```

Comment:
```suggestion

class DietaryHabit(models.Model):
    name = models.CharField(max_length=30)

```
파일 곳곳에 있는 이런 통일성을 해치는 부분 전부 찾아서 수정해주세요~